chart/templates/deployment.yaml

```diff
@@ -23,7 +23,11 @@ spec:
         kubectl.kubernetes.io/default-container: manager
     spec:
       containers:
-      - args: {{- toYaml .Values.controllerManager.manager.args | nindent 8 }}
+      - args:
```

Comment:
Also no need to change this I think. It's already part of the controllerManager args.